### PR TITLE
Refactor muteUnmute in initializePodcastPlayback.js

### PR DESCRIPTION
**Pull request by guntbert**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Simplify toggling of the mute status

## Related Tickets & Documents
xref #3739
## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/javascripts/initializers/initializePodcastPlayback.js [hunk 1]
```diff
@@ -234,19 +234,13 @@ function initializePodcastPlayback() {
...
-      getById('volumeindicator').classList.remove('hidden');
-    } else {
-      audio.muted = true;
-      getById('mutebutt').classList.add('showing');
-      getById('volumeindicator').classList.add('hidden');
-      getById('mutebutt').classList.remove('hidden');
-      getById('volumeindicator').classList.remove('showing');
-    }
+    var mutebutton = getById('mutebutt')
+    var volindicator = getById('volumeindicator')
```
Comment (jacobherrington) [inline app/assets/javascripts/initializers/initializePodcastPlayback.js]: Could these be `const`s instead?

Comment (rhymes) [reply app/assets/javascripts/initializers/initializePodcastPlayback.js]: can they also be camelCase? `muteButton` and `volumeIndicator` for example

Comment (guntbert) [reply app/assets/javascripts/initializers/initializePodcastPlayback.js]: @jacobherrington They indeed **should** :blush:

Comment (guntbert) [reply app/assets/javascripts/initializers/initializePodcastPlayback.js]: @rhymes thx for the guidance :+1:

**Comment by guntbert** [comment]
No worries, the solution from @redxtech is much better anyway :+1:

**Pull request closed without merging**